onnx/test/optimizer_test.py

```diff
@@ -707,6 +707,97 @@ def test_lift_lex_if(self):  # type: () -> None
         assert optimized_model.graph.node[2].attribute[2].strings[0] == b"X"
         assert optimized_model.graph.node[2].attribute[2].strings[1] == b"Y"
 
+    def test_fuse_bn_into_conv_simple_float(self):  # type: () -> None
```

Comment:
Two test cases are so similar, why not merge them together.

Reply:
Agree, should merge to once test function and use a for loop like:

```
for dtype in [TensorProto.FLOAT, TensorProto.DOUBLE]:
  ...
```